languages/clojure/exercises/concept/basic-functions/src/basic-functions.clj

```diff
@@ -0,0 +1,17 @@
+(ns basic-functions)
+
+(defn new-function
```

Comment:
Do the stub functions in this file compile? If not, is there perhaps a feature you can use to have them compile (see [C# example](https://github.com/exercism/v3/blob/master/languages/csharp/exercises/concept/dates/Dates.cs#L7))? The reason I suggest this is that compile errors can be relatively unfriendly to students.

Reply:
These stub functions are valid in Clojure and can be evaluated and called without retuning an error. They'll simply return `nil` until the function body is added.

Reply:
Cool! Thanks for explaining.